lib/endpoints/class-wp-rest-controller.php

```diff
@@ -11,121 +11,121 @@ public function register_routes() {
 	}
 
 	/**
-	 * Get a collection of items
+	 * Get a collection of items.
 	 *
 	 * @param WP_REST_Request $request Full data about the request.
-	 * @return WP_Error|WP_REST_Response
+	 * @return WP_Error|WP_REST_Response.
```

Comment:
Hey, @danielbachhuber !

Periods at the end of the type are the part of standard WordPress PHP documentation in the Core handbook. You can check this information here https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#1-functions-class-methods